### ISSUE TITLE
terraform init fails: Error loading modules: module net_ignition: invalid source string: ../../net/ignition

### ISSUE THREAD

**Issue by lamdor** [coreos/terraform-azurerm-kubernetes#2]
Hi, I just tried out the azurerm tectonic module from the new module registry: https://registry.terraform.io/modules/coreos/kubernetes/azurerm/1.7.3-tectonic.2 and I get a `Error loading modules: module net_ignition: invalid source string: ../../net/ignition` when 

```shell
~/scratch/azure-tectonic-test $ terraform version  
Terraform v0.10.6
~/scratch/azure-tectonic-test $ cat main.tf
module "kubernetes" {
    source = "coreos/kubernetes/azurerm"

    # insert the 7 required variables here
}
~/scratch/azure-tectonic-test $ terraform init
Downloading modules...
Get: https://api.github.com/repos/coreos/terraform-azurerm-kubernetes/tarball/1.7.3-tectonic.2//*?archive=tar.gz
Get: git::https://github.com/coreos/tectonic-installer.git//modules/azure/resource-group?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/azure/vnet?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/azure/etcd?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/azure/master-as?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/azure/worker-as?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/dns/azure?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/bootkube?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/tectonic?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/net/flannel-vxlan?ref=1.7.3-tectonic.2
Get: git::https://github.com/coreos/tectonic-installer.git//modules/net/calico-network-policy?ref=1.7.3-tectonic.2
Error downloading modules: Error loading modules: module net_ignition: invalid source string: ../../net/ignition
```

Is this a terraform bug or a problem with the module source on https://github.com/coreos/tectonic-installer/blob/1.7.3/modules/azure/master-as/ignition-master.tf#L122

**Comment by squat** [comment]
Luke, thanks for reporting this. This is an issue with the module in
tectonic-installer. I'll make a PR ASAP to rectify.

**Comment by squat** [comment]
@rubbish we fixed this issue in the upstream tectonic-installer repo some days ago, however it has not landed in a Tectonic release just yet. We do have a pre-release tag available for 1.7.5, which I have just made available here as well: https://github.com/coreos/terraform-azurerm-kubernetes/releases/tag/1.7.5-tectonic.1-rc.1. Please test this tag to see if it fixes the issue for you.

I'll close this issue once the next Tectonic release is final.

**Comment by squat** [comment]
We have now released 1.7.5+tectonic.1. Please use that tag going forward as it is the latest version of Tectonic and fixes this issue.
